interfaces code fails with incomplete type 'struct sockaddr'

See [build log](http://bot.brew.sh/job/Homebrew%20Pull%20Requests/16702/version=mountain_lion/consoleText).

Failure is related to `darwin/interfaces.cpp`:

```
In file included from osquery/tables/networking/darwin/interfaces.cpp:7:
/usr/include/net/if.h:306:19: error: field has incomplete type 'struct sockaddr'
```

--- a/osquery/tables/networking/darwin/interfaces.cpp
+++ b/osquery/tables/networking/darwin/interfaces.cpp
@@ -5,6 +5,7 @@
 
 #include <ifaddrs.h>
 #include <net/if.h>
+#include <sys/socket.h>
 
 #include <boost/lexical_cast.hpp>
 

